agent/main.go

```diff
@@ -155,3 +167,9 @@ func main() {
 		panic(err)
 	}
 }
+
+func showVersion() {
+	// After Go 1.12, We can use runtime/debug.BuildInfo instead.
```

Comment:
Thanks for linking this.  Seems like https://github.com/golang/go/issues/29814 is probably still a blocker though, as the main module version currently shows as "(devel)" regardless of the state of the repository from which the module was built.

Reply:
Oh, will add the link to the issue then.